Assets/Scripts/Layers/LayerTypes/CartesianTileLayer.cs

```diff
@@ -1,25 +1,23 @@
-using System;
 using UnityEngine;
-using Netherlands3D.CartesianTiles;
 
-namespace Netherlands3D.Twin.UI.LayerInspector
+namespace Netherlands3D.Twin.Layers
 {
     [RequireComponent(typeof(CartesianTiles.Layer))]
     public class CartesianTileLayer : ReferencedLayer
     {
         private CartesianTiles.Layer layer;
         private CartesianTiles.TileHandler tileHandler;
 
-        protected override void OnLayerActiveInHierarchyChanged(bool isActive)
+        public override void OnLayerActiveInHierarchyChanged(bool isActive)
         {
             if (layer && layer.isEnabled != isActive)
                 layer.isEnabled = isActive;
         }
         
-        protected override void Awake()
+        protected void Awake()
         {
-            base.Awake();
-            tileHandler = GetComponentInParent<CartesianTiles.TileHandler>();
+            tileHandler = FindAnyObjectByType<CartesianTiles.TileHandler>();
```

Comment:
What happens here if there are multiple TileHandlers? Because there is no guarantee this is the case? (and yes, the short answer is: it binds to the first one; but isn't that going to give undesired behaviour?)

Reply:
There can be only one ;)

These layers should have a TileHandler and currently when instantiating them when loading one needs to be found, if this is a problem this part can be changed to support multiple TileHandlers through an import adapter that passes an explicit Tilehandler to this class, but I don't see the need for that functionality now

Reply:
We should really have to make a decision on how to move forward with these things; this code will break if someone decides to add a secondary tilehandler. For now I am closing this conversation and leaving as is, but I get more convinced of the need of a Dependency Injection system, Service Locator or some SO that acts as a registry. That should remove implications like this that can backfire in the future